cypress/commons/actions/generic/Scenarios.js

```diff
@@ -179,28 +182,26 @@ function createScenario(scenarioName, isMaster, datasetOrMasterName, runTemplate
 
   getScenarioCreationDialogSubmitButton().click();
 
-  let scenarioCreatedId, scenarioCreatedName;
+  let scenarioDataReq;
   cy.wait('@' + scenarioCreationAlias).then((req) => {
-    scenarioCreatedName = req.response.body.name;
-    scenarioCreatedId = req.response.body.id;
-    cy.wrap(scenarioCreatedId).as('scenarioCreatedId');
-    cy.wrap(scenarioCreatedName).should('equal', scenarioName);
+    scenarioDataReq = req.response.body;
+    expect(scenarioDataReq.name.toLowerCase()).to.equal(scenarioName.toLowerCase());
+    expect(scenarioDataReq.runTemplateName.toLowerCase()).to.equal(runTemplate.toLowerCase());
   });
 
-  cy.wait('@' + scenarioListUpdateAlias).then((req) => {
-    const nameGet = req.response.body.find((obj) => obj.id === scenarioCreatedId).name;
-    cy.wrap(nameGet).should('equal', scenarioCreatedName);
+  return cy.wait('@' + scenarioListUpdateAlias).then((req) => {
+    const nameGet = req.response.body.find((obj) => obj.id === scenarioDataReq.id).name;
+    expect(nameGet).to.equal(scenarioDataReq.name);
+
+    return {
+      scenarioCreatedId: scenarioDataReq.id,
+      scenarioCreatedName: scenarioDataReq.name,
+      scenarioCreatedOwnerName: scenarioDataReq.ownerName,
+      scenarioCreatedCreationDate: new Date(scenarioDataReq.creationDate).toLocaleString(),
```

Comment:
Not a huge fan of hard-coding here the function we're using to format dates in the ScenarioManager cards. Could you at least extract it to a separate function `_formatDate` ? We can keep it in this file for now, we'll see later if we have to share it to different modules.